app/src/main/java/space/taran/arknavigator/mvp/model/repo/scores/ScoreStorage.kt

```diff
@@ -0,0 +1,17 @@
+package space.taran.arknavigator.mvp.model.repo.scores
+
+import space.taran.arknavigator.mvp.model.repo.index.ResourceId
+import space.taran.arknavigator.utils.Score
+
+interface ScoreStorage {
+
+    fun contains(id: ResourceId): Boolean
+
+    fun setScore(id: ResourceId, score: Score)
+
+    fun getScore(id: ResourceId): Score?
+
+    fun countScores(): Int
+
```

Comment:
Why do we need this function?

Reply:
It was being used to find a pinning position as a score but now that we have changed. It's no longer needed. It will be omitted